docs/guide/introduction.rst

```diff
@@ -0,0 +1,296 @@
+This creates a command ``!apples`` that we can type into a channel, and the bot will respond with the given amount of applies in a predefined sentence.. A few key features:
+
+- :class:`~ext.commands.Context` refers to the command invocation context - this includes the :attr:`~ext.commands.Context.channel`, command :attr:`~ext.commands.Context.author`, :attr:`~ext.commands.Context.message` and more.
+- Using ``int`` as a parameter type annotation here will instruct the library to attempt the given argument to an :class:`int` type. This means you'll have an integer in the command body and not a string.
+
+For more information on the commands extension and other converters, please reference :ref:`ext_commands_commands`.
+
+.. code-block:: python
+
+    TOKEN = "your bot token here"
+
+This is the final step, you put your bot token here, save and run the file and the bot will start up.
+
+.. note::
+
+    :meth:`~ext.commands.Bot.run` is :ref:`blocking <faq_blocking>`, so any code after it will **not be run** until the bot has been stopped.
+
+.. warning::
+
+    As this is an example, token security is not applied here. However, you should be very careful with your bot token. Keep it in a secure place and only access it when you are starting the bot.
+    See :ref:`securing_token` for more information.
+
+.. _quickstart_newbot_newcog:
+
+Convenience methods
+~~~~~~~~~~~~~~~~~~~
+
+discord.py provides a pair of useful cli tools for getting a fresh workspace up and ready to go. These utilities are:-
+
+- ``newbot``
+- ``newcog``
+
+They can be used like so:-
+
+.. code-block:: shell
+    :emphasize-lines: 1
+
+    $ python -m discord newbot CoolBot
+    > successfully made bot at /your/path/CoolBot
+
+    $ ls CoolBot/
+    > bot.py cogs/ config.py .gitignore
+
+Which will create a basic bot project with an assortment of essentials for getting started. This includes:-
+
+- A basic ``bot.py`` which is prefilled with a basic :class:`~ext.commands.Bot` subclass and functionality for extending it.
+- A ``cogs/`` directory for creating extension files in with Cog functionality (which is optional).
+- A ``config.py`` which acts as your token or general configuration storage.
+- A ``.gitignore`` file so you don't accidentally push the ``config.py`` file to a version control system like GitHub.
+
+There is also ``newcog`` which does the following:-
+
+.. code-block:: shell
+    :emphasize-lines: 1
+
+    $ python -m discord newcog CoolCog CoolBot/cogs
+    > successfully made cog at CoolBot/cogs/CoolCog.py
+
+This tool creates a new file within the passed directory with the provided name, and again this is a blanket implementation of an :ref:`extension <ext_commands_extensions>` file for use with the :meth:`~ext.commands.Bot.load_extension` method.
+
+The file imports the necessary items and defines a Cog class that you can extend, and adds the necessary ``setup`` method for the extension to function correctly.
```

Comment:
Feels unnecessary to mention this, since it's covered further in the extension docs. This would likely only feel relevant if this guide was to be consumed from 1.x to 2.x migarators, not new people.